admin/templates/configuration/bin/qserv-restart.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+{{QSERV_RUN_DIR}}/bin/qserv-stop.sh
+{{QSERV_RUN_DIR}}/bin/qserv-start.sh
+{{QSERV_RUN_DIR}}/bin/qserv-status.sh
```

Comment:
Ok, I switched to conditional restart, I kept the qserv-status.sh. I don't want to redo all the init.d scripts, it was supposed to be a trivial 0.1 SP, low-hanging fruit type thing. I favor practicality over perfection :)